src/device/mod.rs

```diff
@@ -55,4 +67,13 @@ mod test {
         let device: Device = "/tmp/keypair.bin".parse().expect("file device");
         assert!(matches!(device, Device::File(_)));
     }
+
+    #[cfg(feature = "nova-tz")]
+    #[test]
+    fn test_device_nova_tz() {
```

Comment:
How is this test ever going to pass? The test automation isn't going to run on a device with nova-tz support right/

Reply:
It will work on any device. Just the file on /tmp needed

Reply:
And where is this file? If I turn on tests in CI this will not pass right?

Reply:
Right. Same as test above

Reply:
right so this can't be a test because we want to run tests as part of rust CI

Reply:
Actually I'm wrong. these test just test validity of device string parsing. So they both will work on any hardware and don't rely on file existence

Reply:
ah ok.. I was just digging into why the tests were passing with that URL